scala-thoughts.md

```diff
@@ -0,0 +1,8 @@
+
+It is very easy to do recursion with Scala, I like how you have to declare the type when writing a recursive function. It is also very easy to create parameters to functions in Scala.
+
+I don't love the list creation in Scala. I was originally trying to get user input and put it into a list (I later switched to putting it into a string) and I was really struggling to understand what the rules regarding lists are. I would like to learn more about how lists work in Scala, and understand how one can edit a list. However, I did find it easy to work with strings, which was very nice.
```

Comment:
I also don’t like list operations in Scala, and I found the string manipulation to be really easy in Scala as well. In particular, I had issues figuring out how to index into Scala lists. I only figured out that the function "apply" was a "get element at x" instead of applying some transform after finding the shorthand for indexing (the way you do the indexing in Mastermind).